.github/workflows/release-publish.yml

```diff
@@ -0,0 +1,158 @@
+
+name: Publish release
+
+on:
+  push:
+    tags:
+      - nessie-*
+
+jobs:
+  build:
+    name: Publish release
+    runs-on: ubuntu-latest
+
+    # Runs in the `release` environment, which has the necessary secrets and defines the reviewers.
+    # See https://docs.github.com/en/actions/reference/environments
+    environment: release
+
+    steps:
+
+    ### BEGIN runner setup
+    - name: Update /etc/hosts
+      run: echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Set up JDK 8
+      uses: AdoptOpenJDK/install-jdk@v1
+      with:
+        version: '8'
+        targets: 'JDK8_HOME'
+    # Using Python 3.8
+    # Might be worth to bump the version and/or make it configurable in the future
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Set up Maven toolchains.xml
+      run: cp .github/maven/toolchains.xml ${HOME}/.m2/toolchains.xml
+    - name: Cache local pip repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.m2/repository
+          !~/.m2/repository/org/projectnessie
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Cache local NPM repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Cache local Gradle cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+          ~/.gradle/wrapper
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+        restore-keys: ${{ runner.os }}-gradle
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r python/requirements_dev.txt
+        pip install bumpversion
+    ### END runner setup
+
+    # GH doesn't provide just the tag name, so this step strips `/refs/tags/` from `GITHUB_REF`
+    # and provides the output `VERSION`
+    - name: Get the tag name
+      id: get_version
+      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+    # Log changelog (FYI only)
+    - name: Changelog for ${{ steps.get_version.outputs.VERSION }}
+      env:
+        RELEASE_VERSION: ${{ steps.get_version.outputs.VERSION }}
+      run: |
+        git changelog -t ${RELEASE_VERSION} -n -x /dev/null
```

Comment:
is this built intp `git` now or is this only available w/ `git-extras`. Sorry for the dumb question...

Reply:
Good hint! Added a TODO

Reply:
> Added a TODO

how do you mean? the build will fail at this step if that binary isn't available right?